datacenter/dtr/2.4/guides/release-notes.md

```diff
@@ -11,9 +11,13 @@ known issues for each DTR version.
 You can then use [the upgrade instructions](admin/upgrade.md),
 to upgrade your installation to the latest release.
 
-## Version 2.4.2
+## Version 2.4.3 (2018-03-19)
```

Comment:
Why the change in date format?

Reply:
The majority of components (Engine, compose, ...) were already using the format
```
## version (date)
```

UCP and DTR were among a few components that were using a different format. So we're standardizing on this format and moving forward all components should follow the same convention.